src/panels/lovelace/components/hui-marquee.ts

```diff
@@ -41,42 +34,58 @@ class HuiMarquee extends LitElement {
     }
 
     return html`
-      <div>${this.text}</div>
-      ${this._interval
-        ? html`
-            <div>${this.text}</div>
-          `
-        : ""}
+      <div class="marquee">
+        <div
+          class="marquee-inner ${classMap({
+            animating: Boolean(this._animating),
```

Comment:
Is this `Boolean` Casting needed? animating is a boolean

Reply:
Could be undefined right now, but we could init it false and then we don't need the casting